images-dist/modules/com.redhat.osl.bundleinstall/bundle/manifests/logic-operator-rhel9.clusterserviceversion.yaml

```diff
@@ -732,20 +732,20 @@ spec:
                     fieldRef:
                       fieldPath: metadata.namespace
                 - name: RELATED_IMAGE_JOBS_SERVICE_POSTGRESQL
-                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel9:main
+                  value: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel9:1.37.0
```

Comment:
Ok so this should be part of the cutoff and we need ot set it in the .env file before triggering image-dist sync.

@mareknovotny @saumya1singh